client/app/account/billing/confirmTerminate/billing-confirmTerminate.controller.js

```diff
@@ -1,57 +1,76 @@
-angular.module("Billing.controllers").controller("Billing.controllers.TerminateServiceCtrl", function ($stateParams, BillingTerminate) {
-    "use strict";
+angular.module("Billing.controllers")
+    .controller("Billing.controllers.TerminateServiceCtrl", function ($stateParams, BillingTerminate, User) {
+        "use strict";
 
-    this.reasons = ["LACK_OF_PERFORMANCES", "TOO_EXPENSIVE", "TOO_HARD_TO_USE", "NOT_RELIABLE", "NOT_NEEDED_ANYMORE", "MIGRATED_TO_COMPETITOR", "MIGRATED_TO_ANOTHER_OVH_PRODUCT", "OTHER"];
+        this.reasons = [
+            "LACK_OF_PERFORMANCES",
+            "TOO_EXPENSIVE",
+            "TOO_HARD_TO_USE",
+            "NOT_RELIABLE",
+            "NOT_NEEDED_ANYMORE",
+            "MIGRATED_TO_COMPETITOR",
+            "MIGRATED_TO_ANOTHER_OVH_PRODUCT",
+            "OTHER"
+        ];
 
-    this.serviceId = $stateParams.id;
-    this.serviceState = null;
-    this.token = $stateParams.token;
-    this.loading = true;
-    this.terminating = false;
-    this.error = false;
-    this.globalError = null;
+        this.serviceId = $stateParams.id;
+        this.serviceState = null;
+        this.token = $stateParams.token;
+        this.loading = true;
+        this.userLoading = true;
+        this.terminating = false;
+        this.error = false;
+        this.globalError = null;
 
-    this.init = function () {
-        if (!this.token || !this.serviceId) {
-            this.globalError = true;
-            return;
-        }
+        this.init = function () {
+            if (!this.token || !this.serviceId) {
+                this.globalError = true;
+                return;
+            }
 
-        this.loading = true;
-        BillingTerminate
-            .getServiceInfo(this.serviceId)
-            .then((serviceInfos) => {
-                this.serviceInfos = serviceInfos;
-                return serviceInfos.serviceId;
-            })
-            .then((serviceId) =>
-                BillingTerminate.getServiceApi(serviceId, true).then((service) => {
-                    this.serviceState = _.get(service, "resource.state");
+            User.getUser()
+                .then((user) => {
+                    this.USVersion = user && user.ovhSubsidiary === "US";
+                    this.userLoading = false;
                 })
-            )
-            .catch(() => {
-                this.globalError = true;
-            })
-            .finally(() => {
-                this.loading = false;
-            });
-    };
+                .catch((err) => {
+                    console.log("error", err);
```

Comment:
`console.log`

Reply:
Ooops, slipped out with this PR without having finished the job... ^^